src/pages/workspaces/workspace/notebooks/ExportNotebookModal.js

```diff
@@ -14,9 +19,10 @@ import * as Nav from 'src/libs/nav'
 import * as Utils from 'src/libs/utils'
 import validate from 'validate.js'
 
-
+//TODO: deprecate once notebook tab is removed
```

Comment:
You need another blank line here, we have 2 blank lines between imports and the rest of the file.